site/_docs/reference.md

```diff
@@ -1447,10 +1448,10 @@ period:
 | {fn SUBSTRING(string, offset, length)} | Returns a character string that consists of *length* characters from *string* starting at the *offset* position
 | {fn UCASE(string)} | Returns a string in which all alphabetic characters in *string* have been converted to upper case
 | {fn REPLACE(string, search, replacement)} | Returns a string in which all the occurrences of *search* in *string* are replaced with *replacement*; if *replacement* is the empty string, the occurrences of *search* are removed
+| {fn ASCII(string)} | Returns the corresponding ASCII code of a single-character string
```

Comment:
Pls detail what is happening when string is empty or has more than one character. 
For instance [SQL-Server](https://docs.microsoft.com/en-us/sql/t-sql/functions/ascii-transact-sql?view=sql-server-2017) mentions  :
> Returns the ASCII code value of the leftmost character of a character expression.

Some concrete examples would be nice.

Reply:
It may return negative number when meeting Chinese character because of  encoding with UTF-8.